packages/office-ui-fabric-react/src/components/Callout/examples/Callout.Examples.Styles.ts

```diff
@@ -0,0 +1,95 @@
+import { FontSizes, FontWeights, DefaultPalette } from 'office-ui-fabric-react/lib/Styling';
+import { IStyle } from '@uifabric/styling';
+
+interface ICalloutExampleStyle {
```

Comment:
These styles need to go in the same file as the example that uses them, even if that means duplication (though I don't believe all these styles are used for all examples). Also I'd prefer that you not make interfaces for example styles--when using `mergeStyleSets` as recommended in another comment, an interface isn't needed to get accurate typings, so effectively it just adds visual clutter to the file.

Reply:
Got it and updated to mergeStyleSets pattern!